common/persistence/client/fault_injection.go

```diff
@@ -83,19 +83,19 @@ type (
 var defaultErrors = []FaultWeight{
 	{
 		errFactory: func(msg string) error {
-			return serviceerror.NewUnavailable(msg)
+			return serviceerror.NewUnavailable(fmt.Sprintf("serviceerror.NewUnavailable: %s", msg))
```

Comment:
why do we need to do this? the error type is not enough?

Reply:
We might get the error log in java sdk or somewhere downstream. It might not be clear which error was injected initially.